templates/plugin-pulp2-content-routing.epp

```diff
@@ -0,0 +1,8 @@
+<%- | String[1] $path, String[1] $url | -%>
+  ProxyPass <%= $path %> <%= $url %>
```

Comment:
Is it possible to have a pattern of validation, empty line, template code? e.g.

```
<%- | String[1] $path, String[1] $url | -%>

  ProxyPass <%= $path %> <%= $url %>
```

This pattern of declaration would improve readability for me which can often be hard to follow when we are trying to combine ERB + indentation that is designed to render the *final* file properly nested.